crowbar_framework/app/views/installer/upgrades/finish.html.haml

```diff
@@ -0,0 +1,27 @@
+.row
+  .col-lg-12
+    .panel.panel-default
+      .panel-heading
+        %h2
+          = t("installer.upgrades.header", product: branding_config.page_title)
+
+      .panel-plain.wizard
+        = render partial: "nav", locals: { active: :finish }
+
+        .wizard-body.col-lg-9
+          %h2
+            = t(".header")
+
+          %p
+            = t(".description")
+
+      .panel-footer
+        .row
+          .col-lg-3
+            = progress_steps 8, 1, 8
+
+          .col-lg-9.text-right
+            .btn-group{ role: "group" }
+              = link_to icon_tag(:check, t(".continue")), |
```

Comment:
Don't use the `|` character to split up lines. Wrap on commas or extract code into helper.